README.md

```diff
@@ -61,22 +78,19 @@ curl -X PUT --header 'Content-Type: application/json' http://127.0.0.1:8081/serv
           "version": "3"
         }
     ]
-}' 
-```
-
-Searching tags:
-```
-curl -X GET --header "accept: application/json" \
-     "http://127.0.0.1:8081/service/rest/v1/tags?attribute=status%3Dsuccessful&associatedComponent=repo1%3Agr1%3Acomp1%20%3E%3D%201"
+}
 ```
 
 Deleting a tag:
 ```
-curl -X DELETE --header "accept: application/json" "http://127.0.0.1:8081/service/rest/v1/tags/project1-142"
+DELETE http://127.0.0.1:8081/service/rest/v1/tags/project1-142
+Accept: application/json
 ```
 
-TODO: Add API Documentation
-
+# Compatibility
+Minimum supported version of nexus is 3.15.1-01 at the moment. At the moment, we just test plugin with this version 
```

Comment:
is it good to repeat at the moment two times after each other?
maybe we can replace first at the moment with "for now"

Reply:
Done